docs/products/kafka/kafka-connect/concepts/s3-sink-additional-parameters.rst

```diff
@@ -0,0 +1,49 @@
+S3 sink connector naming and data formats
+=========================================
+
+The Apache Kafka Connect® S3 sink connector enables you to move data from Aiven for Apache Kafka cluster to Amazon S3 for long term storage. The following document describes advanced parameters defining the naming and data formats.
+
+
+S3 naming format
+---------------- 
+
+The Apache Kafka Connect® S3 sink connector by Aiven stores a series of files in the specified S3 bucket. By default, each object is named using the pattern:
+
+::
+
+    <AWS_S3_PREFIX><TOPIC_NAME>-<PARTITION_NUMBER>-<START_OFFSET>.<FILE_EXTENSION>
+
+The placeholders are the following:
+
+* ``AWS_S3_PREFIX``: Can be any string and use placeholders like ``{{ utc_date }}`` and ``{{ local_date }}`` to create different files for each date.
+* ``TOPIC_NAME``: Name of the topic to be pushed to S3
+* ``PARTITION_NUMBER``: topic partitions number
+* ``START_OFFSET``: file starting offset
+* ``FILE_EXTENSION``: the file extension depends on the compression defined in the ``file.compression.type`` parameter. The ``gz`` extension is generated when using the ``gzip`` compression.
+
+.. Tip::
+
+    You can customise the S3 object naming and how records are grouped into files, see the `documentation in GitHub on the file naming format <https://github.com/aiven/aiven-kafka-connect-s3>`_ for further details.
```

Comment:
would be nice to have a direct link - https://github.com/aiven/s3-connector-for-apache-kafka#file-name-format . but link check doesn't like github, right?

Reply:
exactly, linkcheck fails